src/containers/main/UpgradeContainer/index.tsx

```diff
@@ -235,6 +235,33 @@ export const UpgradeContainer: FC<IProps> = ({ address, balance }) => {
 		setHideLowBalance(!hideLowBalance);
 	};
 
+	const addToMetamask = async (address: string, symbol: string, token: any) => {
+		if (symbol.length >= 11) {
+			symbol = symbol.slice(4, symbol.length);
+		}
+		//@ts-ignore
+		const url = iconsCoin[token.coin];
+		const decimals = 18;
+		try {
+			// wasAdded is a boolean. Like any RPC method, an error may be thrown.
+			//@ts-ignore
+			await ethereum.request({
+				method: 'wallet_watchAsset',
+				params: {
+					type: 'ERC20', // Initially only supports ERC20, but eventually more!
+					options: {
+						address: address, // The address that the token is at.
+						symbol: `${symbol}x`, // A ticker symbol or shorthand, up to 5 chars.
+						decimals: decimals, // The number of decimals in the token
+						image: `https://app.ricochet.exchange${url}`, // A string url of the token logo
+					},
+				},
+			});
+		} catch (error) {
+			// console.log(error);
```

Comment:
Add some error handling here, console log or use the red pop ups

Reply:
done